modules/virt-creating-vddk-image.adoc

```diff
@@ -22,7 +22,7 @@ $ mkdir /tmp/<dir_name> && cd /tmp/<dir_name>
 
 . In a browser, navigate to link:https://code.vmware.com/home[VMware code] and click *SDKs*.
 . Under *Compute Virtualization*, click *Virtual Disk Development Kit (VDDK)*.
-. Select the latest VDDK release, click *Download*, and then save the VDDK archive in the temporary directory.
+. Select the VDDK version that corresponds to your VMware vSphere version, for example, VDDK 7.0 for vSphere 7.0, click *Download*, and then save the VDDK archive in the temporary directory.
```

Comment:
If "VDDK 7.0" is a selectable item in a list I think it should be marked up as
`*VDDK 7.0* `

Reply:
No, it is not a selectable item in a list. Change request rejected.